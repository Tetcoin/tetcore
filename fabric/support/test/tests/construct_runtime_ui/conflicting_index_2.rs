use fabric_support::construct_runtime;

construct_runtime! {
	pub enum Runtime where
		UncheckedExtrinsic = UncheckedExtrinsic,
		Block = Block,
		NodeBlock = Block,
	{
		System: system::{} = 5,
		Noble1: noble1::{} = 3,
		Noble2: noble2::{},
		Noble3: noble3::{},
	}
}

fn main() {}
