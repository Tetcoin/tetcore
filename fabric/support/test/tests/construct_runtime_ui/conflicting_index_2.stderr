error: Module indices are conflicting: Both modules System and Noble3 are at index 5
 --> $DIR/conflicting_index_2.rs:9:3
  |
9 |         System: system::{} = 5,
  |         ^^^^^^

error: Module indices are conflicting: Both modules System and Noble3 are at index 5
  --> $DIR/conflicting_index_2.rs:12:3
   |
12 |         Noble3: noble3::{},
   |         ^^^^^^^
